kubernetes/keycloak/realm.json

```diff
@@ -34,29 +34,572 @@
+          "containerId": "c8a7123f-af71-42cc-a4bf-56f6067ea3d2"
+        }
+      ],
+      "hmda2-api": [
+        {
+          "id": "579a3bfa-84e2-410c-a3ea-186628c3c0ca",
+          "name": "hmda-user",
+          "composite": false,
+          "clientRole": true,
+          "containerId": "c749d73d-de71-42fa-aff3-44456890e908"
+        },
+        {
+          "id": "9e59a596-b2a3-45d9-92aa-720afbfa4867",
+          "name": "hmda-admin",
+          "composite": false,
+          "clientRole": true,
+          "containerId": "c749d73d-de71-42fa-aff3-44456890e908"
+        }
+      ],
+      "security-admin-console": [],
+      "admin-cli": [],
+      "broker": [
+        {
+          "id": "c4cbd3d7-65a9-4ae1-953b-bef45f83fdd4",
+          "name": "read-token",
+          "description": "${role_read-token}",
+          "composite": false,
+          "clientRole": true,
+          "containerId": "70a210fc-1f8c-4324-b483-2816407f6f4f"
+        }
+      ],
+      "hmda-api": [],
+      "account": [
+        {
+          "id": "744040ff-eb12-4e35-97ea-98e8ac639656",
+          "name": "view-profile",
+          "description": "${role_view-profile}",
+          "composite": false,
+          "clientRole": true,
+          "containerId": "63989b58-df41-4462-b339-055e842c1bb0"
+        },
+        {
+          "id": "b5f0a127-e502-4be2-8c01-a41442f0c021",
+          "name": "manage-account-links",
+          "description": "${role_manage-account-links}",
+          "composite": false,
+          "clientRole": true,
+          "containerId": "63989b58-df41-4462-b339-055e842c1bb0"
+        },
+        {
+          "id": "6d72cac2-8a56-432d-b198-e19108135b94",
+          "name": "manage-account",
+          "description": "${role_manage-account}",
+          "composite": true,
+          "composites": {
+            "client": {
+              "account": ["manage-account-links"]
+            }
+          },
+          "clientRole": true,
+          "containerId": "63989b58-df41-4462-b339-055e842c1bb0"
+        }
+      ]
+    }
+  },
+  "groups": [],
+  "defaultRoles": ["uma_authorization", "offline_access"],
+  "requiredCredentials": ["password"],
   "otpPolicyType": "totp",
   "otpPolicyAlgorithm": "HmacSHA1",
   "otpPolicyInitialCounter": 0,
   "otpPolicyDigits": 6,
   "otpPolicyLookAheadWindow": 1,
   "otpPolicyPeriod": 30,
-  "otpSupportedApplications": [
-    "FreeOTP",
-    "Google Authenticator"
-  ],
+  "otpSupportedApplications": ["FreeOTP", "Google Authenticator"],
   "scopeMappings": [
     {
       "clientScope": "offline_access",
-      "roles": [
-        "offline_access"
-      ]
+      "roles": ["offline_access"]
+    }
+  ],
+  "clients": [
+    {
+      "id": "c749d73d-de71-42fa-aff3-44456890e908",
+      "clientId": "hmda2-api",
+      "surrogateAuthRequired": false,
+      "enabled": true,
+      "clientAuthenticatorType": "client-secret",
+      "secret": "**********",
+      "defaultRoles": ["hmda-user"],
+      "redirectUris": [
```

Comment:
Most of these are to allow for local development, the most important one for our cluster environments is the relative path, `/*`.